collector/collector.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
```

Comment:
I'd recommend having this in `collector/nginx` so we can do: `nginx.NewCollector()`